Hymns/Common/TabView/TabItem.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+import SwiftUI
+
+public protocol TabItem: Identifiable, Equatable {
+
+    associatedtype SelectedLabel: View
+    associatedtype UnselectedLabel: View
+    associatedtype Content: View
+
+    var selectedLabel: SelectedLabel { get }
+    var unselectedLabel: UnselectedLabel { get }
+    var a11yLabel: Text { get }
```

Comment:
Strange naming? ally label instead of a11lyLabel

Reply:
it's not ally, it's a11y. Are you saying a11y is weird?

Reply:
yeah, just seemed like a strange naming to me. But if thats what you want no problem

Reply:
a11y is shorthand for accessibility, so since this label is used for accessibility, it's the accessibilityLabel, or a11yLabel for short

https://whatisa11y.org/